### ISSUE TITLE
Phase banner

### ISSUE THREAD

**Issue by bencullimore** [nhsuk/nhsuk-service-manual-backlog#89]
## What
> GOV.UK have done a lot of work on phase banners (alpha/beta) which many NHS services have adopted. 

GOV issue on the design system backlog https://github.com/alphagov/govuk-design-system-backlog/issues/57

## Why
A discovery piece ran on NHS.UK to understand what user needs the phase banner was meeting. 

We came to the conclusion that there weren't any user needs, only 'product owner' needs. 

## Anything else
> We also heard anecdotal evidence that the term 'alpha/beta' confused users, and in some cases had a detrimental effect on NHS content - where users had strong user needs not to have 'experimental' information.

**Comment by darrenhutton** [comment]
We had a beta banner on NHS login. It caused issues during Apple App Store approval for app based services using NHS login. Even thought the app using NHS login wasn't in beta, the use of NHS login resulted in rejection.

This prompted the removal of our banner, which looked like this:
![2019-05-28_11-42-21](https://user-images.githubusercontent.com/4036125/58473296-27f88800-8140-11e9-8d97-3e3dcd89f8ef.jpg)
We questioned what the banner was doing for users. Apart from the link to T&Cs we didn't see a benefit.

We could see the benefit of a banner that provided feedback, but we had that elsewhere in the page.

Here's another example of one in use, again no real user benefit:
![2019-05-20_15-13-12](https://user-images.githubusercontent.com/4036125/58473450-89b8f200-8140-11e9-9c40-10bc9204eccc.jpg)

**Comment by davidhunter08** [comment]
From @GrilloPress:

I think we have to be very cautious about things like phase banners.

In health context, trust is key.

And though in beta we may be a little less polished than live, we still have the same standards of clinical assurance. Beta can undermine that when it shouldn’t

Same with “beta” data services. There will be the same standards of data security.

We shouldn’t be undermining people’s confidence in our services. Especially in the healthcare context.
And if beta means “we’re learning and will change things to meet user need” well shouldn’t we always be doing that anyway?

**Comment by StephenHill-NHSBSA** [comment]
I am very much of the mindset that unless it meets a user need - phase banners tend to add very little overall. However there are potentially times when they might.

For example, I work on service which has more than one version of the service running at the same time i.e. a purely offline application (live), and one that only certain groups can apply online for the time being (private beta).

Users obviously need to understand the 2 versions, and which route suits them best. For this use case, the phase banner may come in handy.

**Comment by jackiemellorbrownlee** [comment]
I was recently assessing an NHS service and came to this page to find guidance. It felt like I'd come to the right place, but I was none the wiser about what I should be advising to the designer.

It felt unfair to comment in the assessment about the banner use, so I left it out. But it would be great to get a definitive decision on here, about whether we do banners or not.

**Comment by promisingict** [comment]
> I was recently assessing an NHS service and came to this page to find guidance.
> 
> It felt unfair to comment in the assessment about the banner use, so I left it out. But it would be great to get a definitive decision on here, about whether we do banners or not.

I agree.

The beta phase banner is in use at the moment on several heavily-used, COVID-related NHS services, such as reporting lateral flow test results or ordering LFT kits. It is the GDS phase banner component that has been used on these service pages.

In the absence of an NHS-specific component, is the expected approach that an available GDS component is used in the interim, an example being the Phase banner component?
The [Ministry of Justice design system pages](https://design-patterns.service.justice.gov.uk/components/) make it very clear that "If a similar component is in the GOV.UK Design System, you should use that component instead."

**Comment by sarawilcox** [comment]
@promisingict. Thanks for your comment. We're keen to firm up guidance on phase banners but aren't in a position to do so yet. 

Generally we encourage people to start with GOV.UK components, if there isn't anything in the NHS design system. But if your user research suggests that you need something else in a health context (like the examples above), it is OK to do something different. 

If you're doing any work on this, please feedback and share learnings.

**Comment by sarawilcox** [comment]
I raised the issue of the phase banner with the DHSC assurance team as it came up in a recent service manual enquiry. 

They fed back a general view, plus some specific comments from the team. 

The general consensus in assurance was: 
- robust research and evidence would be needed for any changes 
- it's a good idea to have an indication that the thing is a work in progress of some form 
- there has to be a feedback mechanism 

An adapted version of the end of service page gets more feedback than the standard banner. They haven't seen anyone change the BETA icon, though some teams have changed the wording on the banner.

**Comment by wellhairy** [comment]
@JLarbie and I have recently done some work over in TEL to look at making the phase banner better meet user needs 

We believe the problem that the phase banner is trying to solve is likely to:

- Inform users the service is still being worked on.
- And, to engage and learn from user, to gather feedback and make improvements

This work is intended to pull then focus of this element more towards user engagement. We've updated the term beta to “Feedback”. We believe that this is a much clearer message and will be more widely understood. Feedback is also actionable so users know whats expected at a glance and this follows a very similar pattern to a something put forward on the [alpha gov community channel](https://github.com/alphagov/govuk-design-system-backlog/issues/57#issuecomment-845099555). We've also amended the text to link to the gov.uk beta page to help inform users what this means where appropriate.

### Why

- The term beta isn't very well understood and, likely, may not be understood by some users
- Identifying the phase of development isn't a user need so pulling to the focus towards engaging user and allowing them to feedback will give users a way of effecting change.
- If we need to inform user of a phase of work we should help user understand it means.

### Initial findings

we surveyed 21 users:

- Our research shows that 7 people identified this area as an opportunity to provide feedback and for improvements.
- we asked users about their expectations of what they could do. 15 users reported that they could carry out an action that would feedback. This indicates more confidence in following the call to action.
- By prefacing with the feedback and then asking about the term BETA we saw that individuals correctly identified BETA as a trial phase, however there was no confident link to providing feedback in this process.
- It is possible that the term ‘BETA’ assumes too much knowledge on the user’s part about the Agile phases of delivery, which is not common knowledge to all users.

In summary it would appear that there is significant evidence to suggest that using the primary term ‘Feedback’ to lead the introduction of the Beta banner phase description may be beneficial in strengthening the user’s association of the process of giving feedback and improvements at this point of a website’s lifecycle.

**Comment by frankieroberto** [comment]
NHS Business Services Authority seems to use phase banners on 3 of their services:

* [Get healthcare cover for travelling abroad](https://overseas-healthcare.nhsbsa.nhs.uk/get-healthcare-cover-travelling-abroad/start)
* [Learning Support Fund](https://overseas-healthcare.nhsbsa.nhs.uk/get-healthcare-cover-travelling-abroad/start)
* [Buy an NHS Prescription Prepayment Certificate](https://buy-prescription-prepayment-certificate.nhsbsa.nhs.uk/start)

<img width="1004" alt="Screenshot of Get healthcare cover for travelling abroad header" src="https://github.com/user-attachments/assets/20679e7b-8b3a-4275-9a9e-edcbb6ec108b">

<img width="999" alt="Screenshot of Learning Support Fund header" src="https://github.com/user-attachments/assets/4daecde2-3fba-4c07-8605-245f400845f4">

<img width="996" alt="Screenshot of Buy an NHS Prescription Prepayment Certificate" src="https://github.com/user-attachments/assets/e581fa2b-97d5-41bf-8913-9ecc20f2fbb4">

**Comment by sarawilcox** [comment]
[A proposal for more Service Standard transparency by using the GOV.UK service footer](https://vickytnz.github.io/govuk-header-footer/) by Vicky Teinaki.

**Comment by sarawilcox** [comment]
The GOVUK design system is apparently considering removing their phase banner, as:

- 'beta' is often not understood by users
- it takes up valuable space near the header, especially if you have both an account nav and a service nav
- click-through rates to give feedback can be very low

But, if you don't use it, consider what other ways you could use to make it clear how users can give you feedback.

**Comment by JessDesignsStuff** [comment]
The Prescription Tracker is a service for NHS clinicians and admin staff to find prescription information that is not in their Patient Medication Record (PMR) system. 

When we went into private beta, there was a link to a feedback survey in the header. There was an initial uptake in users submitting feedback, however submissions slowed down after a few weeks. To encourage more feedback, we designed a phase banner with a link to the feedback form, similar to the GOV.UK design.

<img width="1475" height="967" alt="Image" src="https://github.com/user-attachments/assets/16aab73b-21e8-4e95-97ce-1ad35682e256" />
